README.md

```diff
@@ -3,3 +3,31 @@
 In this repo we will try to teach an abstract representation of the world (we referr to this as the *dream*) to our agent and then
 try to make it learn inside the dream.
 
+## Grund integration and minimal documentation
+
+The environment, which we are trying to learn is Match from grund.
+
+To access it, grund has to be installed:
+
+```
+pip install https://github.com/csxeba/grund
```

Comment:
If we use Python3 it should be `pip3 install git+https://github.com/csxeba/grund.git`